Instruments/python/merging_trees.py

```diff
@@ -0,0 +1,41 @@
+import uproot
+import numpy as np
+from array import array
+import ROOT 
+
+usage = 'merging_trees.py'
+ 
+import argparse
+parser = argparse.ArgumentParser(description='Merging two TTrees.')
+parser.add_argument('--files', required=True, type=str, help="Input the files names separated by a comma")  
```

Comment:
It will be easier for scripts to have input files as separate arguments and without specifying --file before each.
Put it as the last parser.add_argument and define it like this:
```python
parser.add_argument('files', type=str, nargs='+', help="Input file names")
```